HEN_HOUSE/doc/src/pirs898-egs++/egs_kerma.doxy

```diff
@@ -0,0 +1,562 @@
+        ...
+    :stop calculation geometry:
+    ...
+    :start calculation geometry:
+        geometry name  = name_n
+        ...
+    :stop calculation geometry:
+
+    correlated geometries = name_i name_j
+    ...
+    correlated geometries = name_k name_l
+
+    ############################################
+    # Fluence scoring requested in this block
+    # Common to all calculation geometries
+    ############################################
+    :start fluence scoring:
+        minimum energy = Emin
+        maximum energy = Emax
+        number of bins = N
+        scale          = linear # linear or logarithmic
+    :stop fluence scoring:
+
+    ######################################
+    # E*muen file, it could also be E*mutr
+    ######################################
+    emuen file = absolute_file_name
+
+    ########################################################
+    # Geometry for forced-detection, if omitted, an analog
+    # scoring used.
+    ########################################################
+    cavity geometry = cavity # For forced detection
+
+:stop scoring options:
+\endverbatim
+
+The meaning of the various keys defining a calculation geometry should
+be self explanatory. The reason one must specify the cavity mass is that
+due to the generality of the geometry package it is not possible to
+automatically compute the cavity volume and therefore the user must supply
+this information for proper normalization. Geometrical regions spanning the
+scoring volume (cavity) are provided via the
+<b><code>cavity regions</code></b> key. If no valid region is entered or
+this input is ignored, no kerma to the cavity of the calculation geometry
+is calculated.
+
+\section egs_kerma_calculation Kerma scoring
+
+Kerma calculations require a file containing pairs of energy \f$ E \f$ and
+\f$ E \times \frac{\mu_{en}}{\rho} \f$ values for the medium of interest.
+This allows a faster evaluation of the
+summation over all contributing particles by saving the multiplication operation.
+The full name (absolute path) of this file <b>must</b> be provided using the
+<b><code>emuen file</code></b> key as shown in the above example.
+
+Kerma for medium \f$m\f$, \f$K_m\f$ is computed by summing up the individual contributions
+of N photons crossing the scoring volume \f$V\f$ using the expression
+
+\f[
+K_m=\sum_{i=1}^{N} w_i \cdot \frac{l_i}{V} \cdot E_i \cdot
+\left(\frac{\mu_\mathrm{en}}{\rho}\right)_i^m
+\f]
+
+The example file <em>emuen_icru90_1.5MeV.data</em> contains \f$ E \times \frac{\mu_{en}}{\rho} \f$
+values for air on a logarithmic energy scale from 1 keV up to 1.5 MeV. These values
+were calculated with the <em>g</em> application using the following (non-default)
+transport parameters:
+
+\verbatim
+##############################
+ :Start MC Transport Parameter:
+
+ Global ECUT=                    0.512
+ Global PCUT=                    0.001
+
+ Photon cross sections  = mcdf-xcom # XCOM with renormalized PE xsections
```

Comment:
Why mcdf? For an example should probably use default transport parameters unless there's a reason otherwise.